src/app/feed/feed.component.scss

```diff
@@ -11,8 +11,8 @@
       cursor: pointer;
       .face {
         vertical-align: middle;
-        width: 64px;
-        height: 48px;
+        width: 128px;
```

Comment:
Since I guess this change is temporary, we should add a comment so we don't forget.